tasks.py

```diff
@@ -0,0 +1,48 @@
+import json
+from typing import Dict
+import datetime
+
+
+def load_records(filepath):
+    with open(filepath) as user_records:
+        loaded_records = json.load(user_records)
+
+    return loaded_records
+
+
+class FullNameCreator():
+    def create_full_name(self) -> Dict:
```

Comment:
So perhaps the function name is a bit misleading here? Not sure though. Does it create a full name? What it seems to do is process `loaded_records` and adds a `full_name` field to each record.

It's great to see you are setting a return type, but I am wondering, does this function really return a `Dict` or is it a `List[Dict]` (a list of dictionaries) ?

Also, even better might be a `TypedDict` instead of a `Dict` that will allow you to define the expected fields (see https://docs.python.org/3/library/typing.html#typing.TypedDict)

So perhaps your return signature might even be something like `List[UserWithFullName]` ?

Reply:
I think you might be right about it returning a List[Dict], that makes sense.

I'm going to consider the name-- FullNameMerger or FullNameFieldCreator might be more straightforward?